scripts/cargo-install-all.sh

```diff
@@ -17,6 +17,9 @@ fi
 
 installDir="$(mkdir -p "$1"; cd "$1"; pwd)"
 cargoFeatures="$2"
+buildProfile="$3"
+[ -n "$buildProfile" ] && binTargetDir="release" || binTargetDir="debug"
```

Comment:
Please expand this out to `if [[ .. ]]; then .. else .. fi`.   More verbose but also more readable